examples/never-type/never_type.description

```diff
@@ -0,0 +1,4 @@
+// The `never` type represents the type of values that never occur.
+// This can be useful to describe the return type of a function, if the function never returns.
+// You can use `never` type as a type parameter. As no value can ever belong to `never` type, you
+// can not declare or assign any value to it.
```

Comment:
How about, "As no value can ever belong to `never` type, variables of type `never` cannot be declared".

Reply:
okay